README.md

```diff
@@ -28,6 +28,22 @@ This automated workflow helps streamline the process of addressing issues and in
 
 To trigger the bot's action, you need to mention "@aiderbot" in the issue title or body, or in the pull request review comment. This ensures that the bot only responds when explicitly called upon.
 
+## Celery Task Queue
```

Comment:
@Aiderbot I don't think we need this much detail. I think we only need a couple of sentences.